common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerErrorResult.java

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.identity.common.internal.broker;
```

Comment:
Minor: This class will need license info

Reply:
Fixed it